SubstrateSdk.podspec

```diff
@@ -8,7 +8,7 @@
 
 Pod::Spec.new do |s|
   s.name             = 'SubstrateSdk'
-  s.version          = '4.0.3'
+  s.version          = '4.0.4'
```

Comment:
WDYT of bumping minor (`4.1.0`)? I think we should prevent automatic updates on patch increment